tests/checks/complete_cygwin.fish

```diff
@@ -0,0 +1,72 @@
+#RUN: fish=%fish %fish %s
```

Comment:
I think we use "-" instead of "_" for most test filenames here,
complete_directories.fish is an exception that should be fixed IMO.